db/controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { retrieveTopics } = require('../models/topics.model')
+
+function getAllTopics(req, res){
+
+    retrieveTopics().then((result) => {
+        res.status(200).send(result)
```

Comment:
We need to send back responses on an appropriate key on the response body. It needs to make sense to the client. For example it should be `.send({topics: result})`  BUT ideally result should be called topics then you could do this `.send({ topics })`.

The test for this endpoint should therefore be asserting that body.topics is the array and it looks as it should

Reply:
Also don't forget catchblocks!